spec/models/user_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  describe "validations" do
+    it { should validate_presence_of :name }
+    it { should validate_presence_of :email }
```

Comment:
Move line 6 right above other `email` validation (keep together for readability)